src/sketchup-stl/exporter.rb

```diff
@@ -269,11 +269,13 @@ def self.do_options
         window_options = {
           :title           => STL.translate('STL Export Options'),
           :preferences_key => PREF_KEY,
-          :height          => 160,
-          :width           => 290,
+          :height          => 170,
+          :width           => 300,
           :modal           => true
         }
         window = SKUI::Window.new(window_options)
+        window.background_color = Sketchup::Color.new(240, 240, 240)
```

Comment:
Can we deal with the background color in another PR?
At the moment all colors are based on system colors, using system color codes in the CSS stylesheet. I'd rather find a solution where we don't have a mix of hard coded and system colors.